CleverPush/CleverPush/Source/CleverPushInstance.m

```diff
@@ -2026,6 +2047,17 @@ - (BOOL)hasSubscriptionTopics {
     return subscriptionTopics ? YES : NO;
 }
 
+- (void)addSubscriptionTopic:(NSString*)topicId callback:(CPAddRemoveTopicHandler)callback {
+    addRemoveTopicResult = callback;
+    NSMutableArray *topics = [[NSMutableArray alloc] initWithObjects:nil];
+    [topics addObject:topicId];
+    [self setSubscriptionTopics:topics];
```

Comment:
Please test how it behaves with multiple calls at the same time to `addSubscriptionTopic`